enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1972,6 +1972,9 @@ func (ws *workspace) config(ctx context.Context) error {
 		// Disable automatic gc - it can interfere with running `rm -rf .git` in
 		// the case where we don't sync successfully.
 		{"gc.auto", "0"},
+		// Disable running gc in the background process.
+		// We want the gc process to finish before snapshotting to save disk space.
+		{"gc.autoDetach", "false"},
```

Comment:
I think it'd be a little more clear to run `git gc --auto --no-detach` instead of setting this config - wdyt?

Reply:
I don't have a strong opinion here. Let's go with the flag approach as you suggested.